tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature

```diff
@@ -0,0 +1,21 @@
+@api
+Feature: create groups, group names are case insensitive
+
+  Scenario Outline: group names are case insensitive, creating a groups with different upper and lower case names
+    Given using OCS API version "<ocs_api_version>"
+    And group "<group_id1>" has been created
+    When the administrator creates a group "<group_id2>" using the GraphApi
+    And the administrator creates a group "<group_id3>" using the GraphApi
+    Then the HTTP status code of responses on all endpoints should be "500"
```

Comment:
is `500` really the reply we want?

Reply:
`500` is the actual current behavior of graph API for creating an existing user or group, as mentioned in comment https://github.com/owncloud/ocis/issues/3516#issuecomment-1098952180 the expected code must be `4XX`
We have an issue reported here https://github.com/owncloud/ocis/issues/3516 and mentioned the behavior in https://github.com/owncloud/ocis/issues/3247.